test/vit-based.jl

```diff
@@ -1,12 +1,10 @@
 using Metalhead, Test
 using Flux
 
-@testset "ViT" begin
-  for mode in [:small, :base, :large] # :tiny, #,:huge, :giant, :gigantic]
+@testset "ViT" begin for mode in [:small, :base, :large] # :tiny, #,:huge, :giant, :gigantic]
     m = ViT(mode)
     @test size(m(x_256)) == (1000, 1)
     @test gradtest(m, x_256)
     GC.safepoint()
     GC.gc()
-  end
-end
+end end
```

Comment:
Seems like a bug. Can you override this?

Reply:
The formatter keeps setting it back. I'll open an issue upstream

Reply:
You are running the formatter locally, right? Can you just fix this manually and we merge this without running the formatted again?

Reply:
Yep, I can do that. But it makes it annoying for future PRs (especially newer contributors) to have to fix this manually every time. I'll defer to whatever you guys decide, though

Reply:
What do you mean? We don't have github actions enforcing the formatting

Reply:
Oh gotcha. I was thinking of formatting the entire project directory but I realised I can just format the `src` directory without formatting `test`. Once #164 lands, I'll add a section to the contributor's guide noting the style enforcement in the `src` folder. I think for `test` it can be enforced by reviewers (for future PRs)